src/ComponentFolders/LoginPage/_test_/Login.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Login from "../Login";
+
+it('renders correctly', () => {
+    const loginComponent = renderer
+        .create(<Login />)
+        .toJSON();
+    expect(loginComponent).toMatchSnapshot();
+});
```

Comment:
Test snapshot has not been added, the test doesn't pass because there is no snapshot :D. A new file should be added with the snapshot.

Reply:
In my code the __snapshots__ folder is already created, I will push it again

Reply:
alright thank you :D